pkg/signal/signal.go

```diff
@@ -11,6 +11,11 @@ import (
 	"syscall"
 )
 
+const (
+	// DefaultStopTimeout is the timeout (in seconds) for the syscall signal used to stop a container.
+	DefaultStopTimeout = 10
```

Comment:
I see. Probably best to leave the default off the CLI argument.
*Might* need to make `StopTimeout` a pointer so we know the difference between 0 and not set... but maybe 0 doesn't make sense here anyway.